instrumentation/akka-http-2.11_2.4.5/build.gradle

```diff
@@ -21,18 +21,21 @@ dependencies {
 
 verifyInstrumentation {
     passesOnly 'com.typesafe.akka:akka-http-experimental_2.11:[2.4.5,)'
-    passesOnly 'com.typesafe.akka:akka-http-experimental_2.10:[2.4.5,)'
-    passesOnly 'com.typesafe.akka:akka-http_2.10:[2.4.5,)'
-    passesOnly('com.typesafe.akka:akka-http_2.11:[2.4.5,)') {
+    passesOnly('com.typesafe.akka:akka-http_2.11:[10.0.0,)') {
         compile("com.typesafe.akka:akka-stream_2.11:2.5.19")
     }
-    passesOnly('com.typesafe.akka:akka-http_2.12:[2.4.5,)') {
```

Comment:
I removed the Scala 2.10 verifier checks from the old instrumentation module because there are no versions >= `2.4.5` on (akka-http-experimental)[https://mvnrepository.com/artifact/com.typesafe.akka/akka-http-experimental] or [akka-http](https://mvnrepository.com/artifact/com.typesafe.akka/akka-http) thus they were never being verified against.

I also updated the start range for Scala 2.11 and 2.12 because `10.0.0` is the [earliest version available](https://mvnrepository.com/artifact/com.typesafe.akka/akka-http), not `2.4.5`.